README.md

```diff
@@ -99,7 +99,15 @@ cd catkin_ws
 source devel/setup.bash
 ```
 
-Now you can launch the software. 
+Now you can launch the bringup package that brings up the launch file for each package:
+
+```bash
+roslaunch marimbabot_bringup marimbabot.launch
+
+# or launch a single package
```

Comment:
I would consider adding a new subsection for the subcomponents instead of adding a comment.

Reply:
k